tests/test-naughty-legacy.lua

```diff
@@ -563,120 +548,86 @@ table.insert(steps, function()
     return true
 end)
 
--- Test more advanced features than what notify-send can provide.
-if has_cmd_send then
-
-    local cmd = { 'dbus-send',
-        '--type=method_call',
-        '--print-reply=literal',
-        '--dest=org.freedesktop.Notifications',
-        '/org/freedesktop/Notifications',
-        'org.freedesktop.Notifications.Notify',
-        'string:"Awesome test"',
-        'uint32:0',
-        'string:""',
-        'string:"title"',
-        'string:"message body"',
-        'array:string:1,one,2,two,3,three',
-        'dict:string:string:"",""',
-        'int32:25000'
-    }
```

Comment:
Could/should this be left?

Reply:
Could: Yes, of course. But why?
Should: I don't know. What would be a reason for leaving it in?

Reply:
I am not sure, just noticed this.
@Elv13 ?

Reply:
Maybe this could be kept to test `notify-send` as a reference implementation?